GLMakie/src/screen.jl

```diff
@@ -164,7 +164,7 @@ mutable struct Screen{GLWindow} <: MakieScreen
     shader_cache::GLAbstraction.ShaderCache
     framebuffer::GLFramebuffer
     config::Union{Nothing, ScreenConfig}
-    stop_renderloop::Bool
+    stop_renderloop::Threads.Atomic{Bool}
```

Comment:
What is the minimum version of Julia you require? `@atomic` is nicer these days,

Reply:
We still run CI on 1.6, which we may want to change at some point, but right now we still try to support 1.6 as long as it's not too bothersome.